Makefile.toml

```diff
@@ -720,23 +720,20 @@ if ! ${TMUX} ls &>/dev/null ; then
   exit 0
 fi
 
-# Kill other components
+# Kill other components with Ctrl+C/Ctrl+D
 ${TMUX} list-windows -F "#{window_name} #{pane_id}" \
-| grep -v 'kafka' \
-| grep -v 'zookeeper' \
+| grep --invert-match --extended-regexp '(kafka|zookeeper)' \
 | awk '{ print $2 }' \
 | xargs -I {} ${TMUX} send-keys -t {} C-c C-d
 
+# Kill kafka cleanly. Ctrl+C will lose data.
 if [[ -n $(${TMUX} list-windows | grep kafka) ]];
 then
   echo "kill kafka"
   kill_kafka || true
 
   echo "kill zookeeper"
   kill_zookeeper || true
-
-  # Kill their ${TMUX} sessions
-  ${TMUX} list-windows -t risedev -F "#{pane_id}" | xargs -I {} ${TMUX} send-keys -t {} C-c C-d
```

Comment:
This command will kill all windows, and causing `${TMUX} kill-server` below fail with `no server running on /private/tmp/tmux-501/risedev`

Reply:
IIUC, `send-keys C-c C-d` first time will kill the running server, and the second time will kill the tmux window. When all the windows are killed, the tmux server is terminated.